### PR TITLE
[14.0][MIG] product_manufacturer

### PR DESCRIPTION
**Pull request by emagdalenaC2i**

### CODE REVIEW

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM, Thanks

**Review by bodedra — APPROVED** [review]
Functional review. :+1:

**Comment by emagdalenaC2i** [comment]
@pedrobaeza and @dreispt Please, could you review and merge this PR?

**Comment by HviorForgeFlow** [comment]
Hello @bodedra @emagdalenaC2i @LoisRForgeFlow I applied a change here: #681. What do you think about allowing product variants to have their own Manufacturer product info? Can we move #681 forward and apply changes there?

**Comment by emagdalenaC2i** [comment]
@HviorForgeFlow Ok, Once #681 is merged I will update this PR

**Comment by HviorForgeFlow** [comment]
@emagdalenaC2i #681 is already merged

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Awaiting #681

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
@emagdalenaC2i This is the historical module. But there is another repo there: https://github.com/OCA/brand/

**Comment by emagdalenaC2i** [comment]
@OCA/product-maintainers please could you merge this PR?

**Comment by HviorForgeFlow** [comment]
> @OCA/product-maintainers please could you merge this PR?

This would decouple version 13 from 14. 

Could you cherry-pick this commit? https://github.com/OCA/product-attribute/pull/681/commits/f69d18ec057631e014cca4f1c67fde8e1f16392e

I think it is not a big effort. Otherwise I can do it, just let me know.

File: product_manufacturer/__manifest__.py
Comment (HviorForgeFlow) [inline product_manufacturer/__manifest__.py:6]: missing website key

```suggestion
    "summary": "Adds manufacturers and attributes on the product view.",
    "website": "https://github.com/OCA/product-attribute",
```

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

**Comment by bluemix** [comment]
I hope this can be merged soon.

**Comment by pedrobaeza** [comment]
Superseded by #797

**Pull request closed without merging**
